### ISSUE TITLE
Export equations? (feature request)

### ISSUE THREAD

**Issue by benjamin-macadam** [varkor/quiver#82]
This app is fantastically useful. Would it be possible to export labelled equations rather than math-mode diagrams for those who like to reference diagrams using \ref or \cref?

**Comment by varkor** [comment]
Could you give an example? Do you mean that `\begin{tikzcd} ... \end{tikzcd}` would be wrapped inside a `\begin{equation} ... \end{equation}`, or do you mean you have a 1-dimensional diagram that you want to export (e.g. using `\rightarrow` instead of TikZ arrows)?

**Comment by benjamin-macadam** [comment]
Yes -  wrapping up the diagram in a `\begin{equation}\label{-} ... \end{equation}` rather than the math-mode `\[ ...\]`. (And perhaps a text field for the label?)

**Comment by varkor** [comment]
For now, you could write `\begin{equation}\label{-} ... \end{equation}` in LaTeX first, and then paste the diagram inside (unchecking the "Centre diagram" checkbox). Perhaps it would be worth adding this feature to **quiver** itself, but I'm not sure how much value it adds compared to the extra complexity in having more options.

**Comment by benjamin-macadam** [comment]
Totally fair!

**Comment by varkor** [comment]
I'm going to leave this issue open so that others can leave feedback if they also think it would be particularly useful.
